README.md

```diff
@@ -1,48 +1,10 @@
-This repo is used for the pentesting using the [ZAP](https://www.zaproxy.org/)'s [stable docker image](https://www.zaproxy.org/docs/docker/about/)  
+This repo is used for the pentesting using the [ZAP](https://www.zaproxy.org/)'s [stable docker image](https://www.zaproxy.org/docs/docker/about/)
 
+### 1) Preparing VAPT-ZAPConfigs directory
 
-### Building the docker image
-```commandline
-   docker build \
-    -t vapt-tool:latest \
-    -f Dockerfile \
-    .
-```
-
-or you can use the pre-built image directly from [docker hub](https://hub.docker.com/r/techprescient/vapt)
-```commandline
-docker pull techprescient/vapt:latest 
-```
-
-### Running the docker image -
-```commandline
-docker run \
-    -p 8080:8080 \
-    -u 0 \
-    -v <zap-configs-dir-path>:/tmp/ZAPConfigs/ \
-    -v <report-output-dir-path>:/reports/ \
-    -i \
-    -e TARGET_URL="https://your-site.com/" \
-    -e LOGGED_OUT_INDICATOR_REGEX="logout-indicatior-for-your-application" \
-    -e USERNAME="your-username" \
-    -e PASSWORD="password" \
-    -e SWAGGER_JSON_URL="https://your-site.com/api-docs" \
-    vapt-tool:latest
-```
-
-Add any extra env variable in the docker run command, if needed.
-
-For example, if you are using the sample cognito script (discussed later) as the authentication script then 
-`AWS_DEFAULT_REGION` should be passed as env variable in the docker run command.
-
-`Note:` Do __NOT__ change the container directories in the run command. 
-Only change the host machine directories and env variable values.
+The directory should have the same structure like `VAPT-Zap-output` directory.
```

Comment:
I can't find any `VAPT-Zap-output` directory in the repo. Please check the whole read me once again.